src/backoffice/dashboards/performance-profiling/dashboard-performance-profiling.element.ts

```diff
@@ -1,15 +1,85 @@
 import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
 import { css, html, LitElement } from 'lit';
-import { customElement } from 'lit/decorators.js';
+import { customElement, state } from 'lit/decorators.js';
+
+import { ApiError, ProblemDetails, ProfilingResource } from '@umbraco-cms/backend-api';
+import { UmbContextConsumerMixin } from '@umbraco-cms/context-api';
+import { UmbNotificationDefaultData, UmbNotificationService } from '@umbraco-cms/services';
 
 @customElement('umb-dashboard-performance-profiling')
-export class UmbDashboardPerformanceProfilingElement extends LitElement {
-	static styles = [UUITextStyles, css``];
+export class UmbDashboardPerformanceProfilingElement extends UmbContextConsumerMixin(LitElement) {
+	static styles = [
+		UUITextStyles,
+		css`
+			uui-toggle {
+				font-weight: bold;
+			}
+
+			h4 {
+				margin-bottom: 0;
+			}
+
+			h4 + p {
+				margin-top: 0;
+			}
+		`,
+	];
+
+	@state()
+	private _profilingStatus = false;
+
+	private _notificationService?: UmbNotificationService;
+
+	private async _getProfilingStatus() {
+		try {
+			const status = await ProfilingResource.getProfilingStatus();
+			status.enabled === true ? (this._profilingStatus = status.enabled) : (this._profilingStatus = false);
+		} catch (e) {
+			if (e instanceof ApiError) {
+				const error = e as ProblemDetails;
+				const data: UmbNotificationDefaultData = { message: error.message ?? 'Something went wrong' };
+				this._notificationService?.peek('danger', { data });
+			}
+		}
+	}
+
+	constructor() {
+		super();
+		this._getProfilingStatus();
+		this.consumeAllContexts(['umbNotificationService'], (instances) => {
+			this._notificationService = instances['umbNotificationService'];
+		});
+	}
 
 	render() {
 		return html`
 			<uui-box>
 				<h1>Performance Profiling</h1>
+				<p>
+					Umbraco currently runs in debug mode. This means you can use the built-in performance profiler to assess the
+					performance when rendering pages.
+				</p>
+				<p>
+					If you want to activate the profiler for a specific page rendering, simply add
+					<strong>umbDebug=true</strong> to the querystring when requesting the page.
+				</p>
+
+				<p>
+					If you want the profiler to be activated by default for all page renderings, you can use the toggle below. It
+					will set a cookie in your browser, which then activates the profiler automatically. In other words, the
+					profiler will only be active by default in your browser - not everyone else's.
+				</p>
+
+				<uui-toggle
+					label="Activate the profiler by default"
+					label-position="left"
+					.checked="${this._profilingStatus}"></uui-toggle>
```

Comment:
This should not show the value of profiling status but rather show if a specific cookie is enabled. We need to figure out if we are doing this with cookies or not, but I would save a state in localStorage for now if this has been toggled or not.